test-harness/specs/static_examples-AC-1.oas2.txt

```diff
@@ -0,0 +1,42 @@
+====test====
+Given
```

Comment:
I don't understand why we would do this? This seems to be duplicating information from the rest of the file, and I am not sure what benefit it serves? 

Yesterday we talked about simplifying the `==test==` content by making it more human, less gherkin, and avoiding duplication with the test file. This seems to have made it less human, more gherkin, increased duplication, and added some blank lines in for unknown reasons. 

> When prism is started with the --dynamic flag, responses should continue to match expected document structure

Done :)